target/linux/ramips/mt76x8/base-files/etc/board.d/02_network

```diff
@@ -90,6 +90,7 @@ ramips_setup_interfaces()
 		ucidef_add_switch_attr "switch0" "enable" "false"
 		ucidef_set_interface_lan "eth0"
 		;;
+	netgear,r6080|\
 	netgear,r6120)
 		ucidef_add_switch "switch0" \
 			"0:lan:4" "1:lan:3" "2:lan:2" "3:lan:1" "4:wan" "6@eth0"
```

Comment:
Have you checked that the LAN port numbering on the case actually is inverted here as well?

Reply:
Yes, double checked this and it is inverted.